test/runtests.jl

```diff
@@ -43,16 +43,48 @@ end
     @test Extents.union(a, c) === nothing
 end
 
-@testset "intersection/intersects" begin
+@testset "intersection/intersects/contains/within" begin
     a = Extent(X=(0.1, 0.5), Y=(1.0, 2.0))
     b = Extent(X=(2.1, 2.5), Y=(3.0, 4.0), Z=(0.0, 1.0))
     c = Extent(X=(0.4, 2.5), Y=(1.5, 4.0), Z=(0.0, 1.0))
-    d = Extent(A=(0.0, 1.0))
+    d = Extent(X=(0.2, 0.45))
+    e = Extent(A=(0.0, 1.0))
+
```

Comment:
I love the extensive tests ❤️ . However, we might need to add even more, seeing the missed cases for intersects, and also to test the equal in the <=?

It might be useful to comment specific cases where you test something very specific (like an equality edgecase)?